client/task-list/test/payments.js

```diff
@@ -58,6 +58,60 @@ describe( 'TaskList > Payments', () => {
 				).toBe( true );
 			} );
 		} );
+
+		describe( 'Mollie', () => {
+			it( 'Detects the plugin is enabled based on the options passed', () => {
+				const mollieParams = {
+					...params,
+					options: {
+						woocommerce_mollie_payments_settings: {
+							enabled: 'yes',
+						},
+					},
+				};
+
+				const mollieMethod = getPaymentMethods( mollieParams ).find(
+					( method ) => method.key === 'mollie'
+				);
+
+				expect( mollieMethod.isEnabled ).toBe( true );
+			} );
+
+			it( 'is enabled for supported countries', () => {
```

Comment:
Some of these tests seem a bit redundant, but I added them anyway, the upside I can think of is that this will catch accidental regressions or at least make someone think before changing country support for the plugin.

Reply:
Testing all those countries LGTM 👍 

Jest has [`test.each`](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout) to natively loop through a bunch of items.